plugins/helo.checks.js

```diff
@@ -1,9 +1,10 @@
 'use strict';
 // Check various bits of the HELO string
 
-var dns       = require('dns');
-var net_utils = require('./net_utils');
-var utils     = require('./utils');
+var dns       = require('dns'),
+    async     = require('async'),
```

Comment:
Minor nit - we normally declare var for each require() and terminate each line with a semicolon.